src/pages/Home.tsx

```diff
@@ -1,24 +1,36 @@
 import React from 'react';
```

Comment:
Update import and usage below

```suggestion
import React, { useEffect } from 'react';
```